config/game_options.txt

```diff
@@ -77,8 +77,8 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
 ## Set to 0 to disable that mode.
 
 PROBABILITY TRAITOR 5
```

Comment:
Knock traitor down two notches. It already has three different things that are effectively the same gamemode.

Reply:
redundant